ruspy/estimation/est_cost_params.py

```diff
@@ -6,6 +6,7 @@
 import numba
```

Comment:
Could you rename the moduel to nfxp.py . Then it is consistent with mpec.py module.